python/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.5
+
+RUN pip install \
```

Comment:
your README indicates this image is intended for both 2.7 and 3.5, which of those does this `pip` reference, and does that matter for your purposes?

Reply:
I haven't encountered it mattering, but it'll be python 3.

Tox will create virtualenvs for the two versions automagically, so pips therein (to install reqs) will be as appropriate for their virtualenv python version.

One may also manually create a python2 venv (in a Dockerfile that FROMs this one, or a script or what have you):

```
root@2af30f6543c4:~# virtualenv -p python2 .venv
Running virtualenv with interpreter /usr/bin/python2
...
root@2af30f6543c4:~# source .venv/bin/activate
(.venv) root@2af30f6543c4:~# python --version
Python 2.7.9
```

I can update the readme with this information if the current ambiguity is too strong.